app/src/renderer/components/wallet/PageWallet.vue

```diff
@@ -129,13 +129,13 @@ export default {
 </script>
 
 <style lang="stylus">
-main
-  .tm-li-label
-    max-width calc(100% - 110px)
+@require '~variables'
+
+@media screen and (min-width: 640px)
```

Comment:
remove media queries

Reply:
i know it's not the scope of this ticket but since we already know this is the plan - we might as well remove them as opposed to editing them IMO